tests/system/test_system.py

```diff
@@ -149,6 +150,18 @@ def test_hmac_key_crud(self):
 
         before_keys = set(Config.CLIENT.list_hmac_keys())
 
+        now = datetime.datetime.utcnow().replace(tzinfo=_helpers.UTC)
```

Comment:
With `pytz` in the standard library now, which includes a `utc`, is there any need to still use our own handrolled?

Reply:
```bash
$ python3.6
Python 3.6.10 (default, Apr 20 2021, 13:17:17) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytz
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'pytz'
>>>
```
See https://github.com/googleapis/python-api-core/commit/61490f5e9c6f3ba4729e35cd1fc8435c82a43041 (part of https://github.com/googleapis/python-api-core/pull/212), which uses `datetime.timezone.utc`.

GCS is still stuck supporting Python 2.7, however.